cmd/list.go

```diff
@@ -46,7 +47,11 @@ var (
 
 			limit := min(maxResults, len(versions))
 			for _, version := range versions[:limit] {
-				fmt.Println(helpers.ColoredVersion(version))
+				if helpers.IsPreReleaseVersion(version) && includePreReleaseVersions {
```

Comment:
I think this can be captured in a single if statement?

```go
if !helpers.IsPreReleaseVersion(version) || includePreReleaseVersions {
```

Reply:
True, forgot to simplify the expression.  The if else construction is no longer needed since the helper function prints the version with the right color code now 💯 
Resolved in [06248fa](https://github.com/tfversion/tfversion/pull/22/commits/06248faf805808200c86ab1695bae5009e95fb58)